codegen/src/main/scala/akka/grpc/gen/javadsl/play/PlayJavaBothCodeGenerator.scala

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.gen.javadsl.play
+
+object PlayJavaBothCodeGenerator extends PlayJavaServerCodeGenerator with PlayJavaClientCodeGenerator {
+
```

Comment:
Shouldn't we override the name here as well?

Reply:
I'll do that